docs/tools/setup.md

```diff
@@ -2,17 +2,26 @@
 
 To start, you can use the <a href="https://babeljs.io/docs/plugins/preset-env/">env preset</a>, which enables transforms for ES2015+
 
+To make use of Babel 7, one need to use the naming convention as with `@babel` as follows:
```

Comment:
I don't think we need to make the Babel 7 distinction since these docs are specifically for Babel 7+

Reply:
Yeah, it is just a package rename. Also, unofficial packages still use `babel-*`